app/src/pages/common/settingsPage/notificationsTab/notificationCaseList/notificationCaseList.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import classNames from 'classnames/bind';
+import { showModalAction } from 'controllers/modal';
+import { convertNotificationCaseForSubmission } from '../utils';
+import { AddNewCaseButton } from '../addNewCaseButton';
+import { CaseListItem } from './caseListItem';
+import styles from './notificationCasesList.scss';
+
+const cx = classNames.bind(styles);
+
+@connect(null, {
+  showModal: showModalAction,
+})
+export class NotificationCaseList extends Component {
+  static propTypes = {
+    showModal: PropTypes.func,
+    cases: PropTypes.array,
+    readOnly: PropTypes.bool,
+    updateNotificationCases: PropTypes.func,
+  };
+  static defaultProps = {
+    updateNotificationCases: () => {},
+    showModal: () => {},
+    cases: [],
+    readOnly: false,
+  };
+
+  onDelete = (id) => {
+    const { showModal } = this.props;
+    showModal({
+      id: 'deleteNotificationCaseModal',
+      data: {
+        id: id + 1,
```

Comment:
`id + 1`? If you need to somehow format this id, it's better to do it closer to the place you render it - this will be less confusing.

Reply:
Fixed